packages/components/src/ComboBox/docs/ComboBoxOption/autoMenuWidth.tsx

```diff
@@ -1,25 +1,24 @@
-import { ComboBox, ComboBoxOption, ComboBoxOptions, Label, Text } from "@hopper-ui/components";
+import { ComboBox, ComboBoxItem, Text } from "@hopper-ui/components";
 
 export default function Example() {
     return (
         <ComboBox
             aria-label="list of options with a description"
+            isAutoMenuWidth
+            label="Roles"
         >
-            <Label>Roles</Label>
-            <ComboBoxOptions isAutoMenuWidth>
-                <ComboBoxOption textValue="Developer">
-                    <Text>Developer</Text>
-                    <Text slot="description">Builds, tests, and maintains software.</Text>
-                </ComboBoxOption>
-                <ComboBoxOption textValue="Cat">
-                    <Text>Designer</Text>
-                    <Text slot="description">Creates visually appealing, functional solutions.</Text>
-                </ComboBoxOption>
-                <ComboBoxOption textValue="Frog">
-                    <Text>Manager</Text>
-                    <Text slot="description">Responsible for leading and overseeing a team or department to ensure organizational goals are met.</Text>
-                </ComboBoxOption>
-            </ComboBoxOptions>
+            <ComboBoxItem textValue="Developer">
```

Comment:
I'm guessing moving the label and description of the ComboBoxItem from slot to prop will be later?

Reply:
no, I think these will stay the same. If you check the code out here: https://reactspectrum.blob.core.windows.net/reactspectrum/6de1ad4d4bab6d6debacf576b8ba325871064e7e/storybook-s2/index.html?path=/docs/picker--docs#with-icons
They still have it as slots probably because it doesn't fall under the field category. 
What do you think, @alexasselin008 ?